src/endpoints/get-diffs.js

```diff
@@ -22,6 +22,19 @@ module.exports = cors(async (req, res) => {
 
   if (!session) return error(res, 404, "Session Not Found")
 
+  const samplesQueryResults = db.prepare(`SELECT  *
+                           FROM latest_sample_state
+                           WHERE session_short_id = ?`).all(session.short_id);
+
+  const samples = []
+  samplesQueryResults.forEach(sample => {
+    const content = JSON.parse(sample.content)
+    const annotation = JSON.parse(sample.annotation)
+    samples.push(Object.assign({}, content, {annotation}))
```

Comment:
prefer the spread syntax, `{ ...content, annotation }`

Reply:
Cool